System/Posix/Files/Common.hsc

```diff
@@ -940,8 +940,8 @@ pattern StatxBtime = StatxMask 0
 
 -- | Want @stx_mnt_id@.
 pattern StatxMntId :: StatxMask
-#ifdef STATX_MNT_ID
-pattern StatxMntId = StatxMask (#const STATX_MNT_ID)
+#ifdef HAVE_STATX_MNT_ID
+pattern StatxMntId = StatxMask (#const HAVE_STATX_MNT_ID)
```

Comment:
Surely the `#const` value should **NOT** have changed.

Reply:
oops